content/docs/standard/data-resource.mdx

```diff
@@ -275,3 +280,38 @@ An example of a resource with a schema:
   }
 }
 ```
+
+### JSON
+
+The properties below are applicable to any JSON Data Resource.
+
+#### `type` {#json-type}
+
+A JSON Data Resource descriptor `SHOULD` contain a property `type` that `MUST` be `json`.
+
+#### `path` or `data` [required] {#json-path-or-data}
+
+If the `path` property is used for providing data than it `MUST` contain [JSON Data](/standard/glossary/#json-data).
+
+If the `data` property is used for providing data for a JSON Data Resource than it `MUST` be an inline instance of [JSON Data](/standard/glossary/#json-data).
+
+#### `jsonSchema`
+
+A JSON Data Resource `MAY` have a `jsonSchema` property to provide a JSONSchema of the resource data. If provided, the `jsonSchema` property `MUST` be a valid [JSON Schema](https://json-schema.org) descriptor in a form of an object or [URL-or-Path](/standard/glossary/#url-or-path).
+
+An example of a resource with a JSON Schema:
+
+```json
+{
+  "name": "data",
+  "type": "table",
+  "path": "data.json",
+  "jsonSchema": {
+    "type": "object",
+    "properties": {
+      "first_name": { "type": "string" },
+      "last_name": { "type": "string" }
```

Comment:
Could we please follow our own standard and use "given_name", "family_name" instead? (maybe even better if we use JSON-y naming conventions: givenName, familyName)